test/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# Enable CTest support
+enable_testing()
```

Comment:
it allows running tests with `ctest` command. The main benefit of it is ability to parallelize test running, but currently some tests conflict with each other 😄